Windows/ppsspp.rc

```diff
@@ -228,8 +266,26 @@ BEGIN
     CONTROL         "",IDC_GEDBG_MATRICES,"SysListView32",LVS_ALIGNLEFT | LVS_SHOWSELALWAYS | LVS_REPORT | WS_BORDER | WS_TABSTOP,7,7,217,86
 END
 
-
+#ifdef __MINGW32__

```

Comment:
Why does this need to be different on mingw?

Reply:
because the resource compiler doesn't support utf-16.
this is actually not even a good solution since it doesn't even treat the file as utf-8.
I can't figure out a better solution for this so this was the only workaround I could come up with.

Reply:
Oh, darn. But please add a comment saying that, just on the line above.